frontend/app/src/i18n/en.json

```diff
@@ -425,6 +425,7 @@
     "guestUser": "Guest User",
     "halloffame": {
         "allTime": "All time",
+        "balance": "CHIT",
```

Comment:
I know its the same in every language but we should still add the translations, otherwise they'll get added next time we run the script and they may get incorrectly translated.

Reply:
There is a bunch of CHIT related stuff not translated (using sync_lang) yet because it is really placeholder text and it will definitely need deleting from all the translation files which is a pain.